### ISSUE TITLE
Error when using idyll build with output directory set to current directory

### ISSUE THREAD

**Issue by Harmon758** [idyll-lang/idyll#447]
**Description**
An error occurs when using `idyll build` and setting the output directory to the same current directory:
```
C:\Users\User\Desktop\Example>idyll build -o .

Building Idyll project with output directory: .

Error: Source and destination must not be the same.
    at copy (C:\Users\User\Desktop\Example\node_modules\fs-extra\lib\copy\copy.js:36:31)
    at Promise (C:\Users\User\Desktop\Example\node_modules\universalify\index.js:13:12)
    at new Promise (<anonymous>)
    at copy (C:\Users\User\Desktop\Example\node_modules\universalify\index.js:7:14)
    at Promise.all.pathExists.then.exists (C:\Users\User\Desktop\Example\node_modules\idyll\src\pipeline\index.js:105:20)
```

**Reproduction Steps**
1. Open terminal
2. `idyll create` a new Idyll post
3. `cd` to the new directory
4. Enter `idyll build -o .`
4. Wait and see error

**Expected Behavior**
No error with `idyll build -o .` successfully building

**Screenshots:** N/A

**Desktop:**
 - OS: Microsoft Windows 7 (64-bit) [Version 6.1.7601]
 - Browser: N/A
 - Version: 3.12.0

**Smartphone:** N/A

**Additional Context**
This occurs because at the end of the build, it attempts to copy the `static` directory over to the output directory:
https://github.com/idyll-lang/idyll/blob/697fe62e2e97b47be45dfc227d6a51329cd4a306/packages/idyll-cli/src/pipeline/index.js#L105 Since this is unnecessary in this case and occurs at the very end, the build successfully completes, but still outputs this error.

**Comment by mathisonian** [comment]
Thank you for the detailed report @Harmon758. I'm just picking things back up after the holiday break and should have an update for you on this and #446 soon.

**Comment by mathisonian** [comment]
Fixed in #450.
